generator/templates/tupleserializer.gotmpl

```diff
@@ -208,7 +208,15 @@ func ({{.ReceiverName}} *{{ pascalize .Name }}) UnmarshalJSON(raw []byte) error
      return err
    }
    {{ .ReceiverName }}.{{ .GoType }} = {{ varname .Name }}
-   {{ end }}{{ end }}
+   {{ end }}
+   {{ range .Properties }}
```

Comment:
This logic is already defined here: https://github.com/Simon-Li/go-swagger/blob/d61294f01b4d2aac01024432a775db4bb5a22ff7/generator/templates/tupleserializer.gotmpl#L187-L203 the fix for this issue should figure out why that code isn't doing the job.